.travis.yml

```diff
@@ -51,6 +51,7 @@ install:
           $HOME/.cabal/packages/hackage.haskell.org/00-index.tar;
    fi
  - travis_retry cabal update -v
+ - cabal install cpphs
```

Comment:
wait, why do we need cpphs to test this?

Reply:
If this isn't needed anymore, can we remove it? :)